### ISSUE TITLE
windows 下，浏览器访问不存在的网页会显示 EOF 文本，而不会显示网页不存在页面

### ISSUE THREAD

**Issue by AT-Aoi** [getlantern/lantern#5825]
![image](https://user-images.githubusercontent.com/3676278/29249666-c9e9f400-8066-11e7-9e6b-d0babb895547.png)
Vivaldi (Chrome 内核)
![image](https://user-images.githubusercontent.com/3676278/29249678-f964f860-8066-11e7-8477-c4772efec65f.png)
IE 11
![image](https://user-images.githubusercontent.com/3676278/29249710-5f848d90-8067-11e7-9de1-d09dd05e8a33.png)
Firefox

**Comment by yaoqs** [comment]
refresh your page， and after some times u will get the page u want to

**Comment by AT-Aoi** [comment]
@yaoqs No. That page doesn't even exist indeed. I say in this case, lantern handles the request wrongly.
